src/NUnitFramework/TestBuilder.cs

```diff
@@ -179,7 +179,7 @@ public static ITestResult ExecuteWorkItem(WorkItem work)
             // TODO: Replace with an event - but not while method is static
             while (work.State != WorkItemState.Complete)
             {
-#if PORTABLE || NETSTANDARD1_6
+#if PORTABLE
                 System.Threading.Tasks.Task.Delay(1);
 #else
                 Thread.Sleep(1);
```

Comment:
That's fair enough for our framework code itself. For tests, it may be different. I replaced `Thread.Sleep` with `Task.Delay` in some of our tests to eliminate spurious failures. My general impression is that Sleep has more of a tendency to go extra long in some circumstances.